ludwig/features/audio_feature.py

```diff
@@ -357,30 +364,18 @@ def __init__(self, feature):
                 'length has to be defined - '
                 'check "update_model_definition_with_metadata()"')
 
-    def _get_input_placeholder(self):
-        return tf.placeholder(
-            tf.float32, shape=[None, self.length, self.embedding_size],
-            name='{}_placeholder'.format(self.feature_name)
-        )
+    def call(self, inputs, training=None, mask=None):
+        assert isinstance(inputs, tf.Tensor)
+        assert inputs.dtype == tf.float32  # TODO: check that only tf.float32 is correct
+        # assert len(inputs.shape) == 2  # TODO: check correct inputs.shape as per sequence feature
```

Comment:
This should be assert len(inputs.shape) == 2, audio features are either raw, in that case [b, l, 1] or with extracted features, in which case [b, l, f] where f is the number fo features

Reply:
Actually I commented that row because I always got == 3 and wanted to check wether it was correct, since being an extension of the sequence feature I expected to have the same shape lenght (which is 2 in the sequence feature code).